packages/devtools_app/lib/src/shared/diagnostics/inspector_service.dart

```diff
@@ -673,32 +625,6 @@ abstract class InspectorObjectGroupBase
           );
   }
 
-  Future<RemoteDiagnosticsNode?> invokeServiceMethodWithArgReturningNode(
-    String methodName,
-    String arg,
-  ) async {
-    if (disposed) return null;
-    return useDaemonApi
-        ? parseDiagnosticsNodeDaemon(
-            invokeServiceMethodDaemonArg(methodName, arg, groupName),
-          )
-        : parseDiagnosticsNodeObservatory(
-            invokeServiceMethodObservatoryWithGroupName1(methodName, arg),
-          );
-  }
```

Comment:
this method is used by an internal customer. Please revert.

Reply:
Done